### ISSUE TITLE
Can't get the text one to display anything

### ISSUE THREAD

**Issue by llelectronics** [maui-artwork/maui-artwork-plymouth#1]
I can't get the maui-text to work. Is it missing some dependency ? 
Is the module name correct ?

**Comment by UriHerrera** [comment]
It used to work fine maybe it had something to do that the path it's not `/lib/plymouth/themes` anymore?

**Comment by llelectronics** [comment]
Not that I know of as maui-logo is working fine. Maybe it's an issue on my machine only. The new ISO I built with this set as plymouth theme I only got the 3 dots default plymouth text splash. I will play with it around. Seems it does not know to use the text or whatever plugin it should.

**Comment by UriHerrera** [comment]
I'll try with another file. I'll add that I keep getting a warning from Plymouth that `/usr/lib/x86_64-linux-gnu/plymouth/[theme_name].so` doesn't exist and that it's "skipping that theme" that warning doesn't come up with the graphical splash.

**Comment by UriHerrera** [comment]
Well it's working now. https://github.com/maui-artwork/maui-artwork-plymouth/commit/887139bdbb0df8cbc53c826e22785e265e6c0c25

_I don't know why._

![](http://i.imgur.com/OQTJgfk.png)

**Comment by star-buck** [comment]
Note: Its not "Project" any longer, its Maui OS or Maui Linux or simply
MAUI.

**Comment by llelectronics** [comment]
I guess it is the module name coressponding to the so file it should use. Sadly ubuntu-text is only available on ubuntu I fear.

**Comment by llelectronics** [comment]
I guess it is the module name coressponding to the so file it should use. Sadly ubuntu-text is only available on ubuntu I fear.

**Comment by UriHerrera** [comment]
Maybe ship the module too?.

# 

I can also use the `text` module but then it's just the 3 dots and no text.

# 

There's also use the `breeze-text` module although it's on KDE Neon maybe that one can be used —or it's available to use— in other distributions?

_it's available in the AUR_

![](http://i.imgur.com/GdjvVjm.png)

**Comment by llelectronics** [comment]
Yeah we could add the code from neon as we can't ship binary onlys as they might break in future releases. 

In general I am asking myself currently where is this text splash shown anyways nowadays. I think only on virtual machines which have to less memory assigned for the graphics card. 
All other systems would show the splash. So maybe the 3 dots default plymouth text is enough.

**Comment by star-buck** [comment]
Yeah, lets say the 3 dots is enough.
